src/MeetignsSDKAdapter.test.js

```diff
@@ -0,0 +1,42 @@
+import {isObservable} from 'rxjs';
+
+import MeetingSDKAdapter from './MeetingsSDKAdapter';
+import createMockSDK, {mockSDKMeeting} from './__mocks__/sdk';
+
+describe('Meetings SDK Adapter', () => {
+  let mockSDK, meetingSDKAdapter;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    meetingSDKAdapter = new MeetingSDKAdapter(mockSDK);
+  });
+
+  describe('getMeeting() functionality', () => {
+    test('returns an observable', () => {
+      expect(isObservable(meetingSDKAdapter.getMeeting())).toBeTruthy();
+    });
+
+    test('returns a meeting in a proper shape', (done) => {
+      meetingSDKAdapter.getMeeting('id').subscribe((meeting) => {
+        expect(meeting).toEqual(
+          expect.objectContaining({
+            id: mockSDKMeeting.id,
```

Comment:
```suggestion
            ID: mockSDKMeeting.id,
```

However, the shape of the meeting should match that of what is [defined in the interface](https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L8). If the property has no value, then it should be set to `null`